docker_config/dockerfiles/Dockerfile.default-cpu

```diff
@@ -13,21 +13,15 @@ RUN apt-get update && apt-get install -y \
     apt-transport-https \
     iputils-ping \
     git \
-    python2.7 \	
-    python-pip \	
-    python-dev \		
-    python-tk \
+    curl \
     build-essential \
     cmake \
     libhdf5-dev \
     swig \
-    wget \
-    curl
+    wget
 
 ## Python 3.6
-RUN add-apt-repository ppa:deadsnakes/ppa && \
-    apt-get update -y  && \
-    apt-get install python3.6 -y \
+RUN apt-get install python3.6 -y \
         python3.6-venv \
         python3.6-dev \
         python3-pip \
```

Comment:
At the end, we should also add `&& rm -rf /var/lib/apt/lists/*`, which will save more space -- see https://stackoverflow.com/questions/61990329/dockerfile-benefits-of-repeated-apt-cache-cleans. let's also add this to the end of all apt-get install commands.